src/app/dag-state.service.ts

```diff
@@ -16,6 +16,7 @@
  */
 
 import {ElementRef} from '@angular/core';
+import * as lodash from 'lodash';  // from //third_party/javascript/lodash:lodash-module
```

Comment:
This is a pretty expensive dependency to add for this purpose, however if you need to add it, you could just import that specific function
```suggestion
import isEqual from 'lodash/isEqual';  // from //third_party/javascript/lodash:lodash-module
```